src/admin/class-wordlift-admin-entity-list.php

```diff
@@ -80,4 +80,73 @@ public function render_custom_columns( $column, $entity_id ) {
 		}
 		
 	}
+	
+	/**
+	 * Add 4W select box before the 'Filter' button.
+	 *
+	 * @since 3.3.0
+	 */
+	public function add_4W_filter() {
+		global $typenow;
+		
+		// Only show on entity list page
+		if( $typenow !== Wordlift_Entity_Service::TYPE_NAME ){
```

Comment:
the method in `Wordlift_Entity_Service` is for a single entity:
```
public function is_entity( $post_id ) {

		return ( self::TYPE_NAME === get_post_type( $post_id ) );
	}
```
in this case the check is on the post_type of all the elements in the page, so no specific ID.